content/04.study.md

```diff
@@ -47,6 +47,25 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Introduction 
+
+DNA Methylation (DNAm), is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
+This DNA-level epigenetic modification regulates gene transcription, is critical in development, and alterations to DNA methylation are well-established as contributing to pathophysiology of many diseases including cancers. 
+Studies of DNA methylation have also shed light on the processes involving initial differentiation of stem cells, aging and pathogenesis in response to environmental exposures. 
+
+Traditional analytic approaches to DNA methylation data often focus on estimating differential DNA methylation between groups or related with an outcome using linear mixed effects models. 
+In addition, a growing application of DNA methylation measures is to infer cellular or subject phenotypes from samples and either examine the relation of these phenotypes with outcomes or disease states directly, and/or include them in models as covariates. 
+For example, inference of subject age using DNA methylation clock approaches are  established [@tag:Horvath2013], and are starting to be applied to test the relation of biological age with disease risk and outcomes [@tag:Kresovich2019]. 
+Importantly, reference DNA methylation profiles of leukocytes are available to infer immune cell type proportions with measures of DNA methylation in peripheral blood DNA [@tag:Houseman2012],[@tag:Salas2018], a novel approach to immunophenotyping that only requires. 
```

Comment:
```suggestion
Different cell types have different DNA methylation profiles.
Reference DNA methylation profiles of leukocytes are available, which makes it feasible to infer immune cell type proportions with measures of DNA methylation in peripheral blood DNA [@tag:Houseman2012],[@tag:Salas2018], a novel approach to immunophenotyping that only requires. 
```
☝️ missing word at the end of this sentence. I also added a short sentence ot introduce the idea that DNA methylation differs between cell types to help the computational reader along.